lib/ansible/playbook/handler.py

```diff
@@ -35,9 +41,21 @@ def __repr__(self):
         return "HANDLER: %s" % self.get_name()
 
     @staticmethod
-    def load(data, block=None, role=None, task_include=None, variable_manager=None, loader=None):
-        t = Handler(block=block, role=role, task_include=task_include)
-        return t.load_data(data, variable_manager=variable_manager, loader=loader)
+    def load(data, block=None, role=None, task_include=None, variable_manager=None, loader=None, play=None):
+        h = Handler(block=block, role=role, task_include=task_include)
+        h.load_data(data, variable_manager=variable_manager, loader=loader)
+        if h.get_name() == 'include':
```

Comment:
you might want to get the module/action as the include can be given a name that does not match `include:`